### DISCUSSION TITLE
Simplify minting experience

### DISCUSSION THREAD

**Discussion by roiLeo** [kodadot/nft-gallery#3898] [General]
#### Ideas that come to my mind
- [ ] limit number of nfts
- [ ] add link to current collection
- [ ] option to mint multiple nfts in one collection (mass mint?)
- [ ] don't redirect to nft page if user don't want to (in order to mint another item)
- [ ] prefil traits with already minted nfts in collection (will help to create some stats %)
- [x] uncheck "LIST for sale" by default
- [ ] update royalty slider with prefilled options (25%, 50%, 75%, otherNft%)
- [ ] markdown editor for "Description" textarea
- [ ] show/hide on-chain attribute is useless
- [ ] required attributes fields
- [ ] carbonless???
- [x] full width submit button

**Comment by yangwao** [comment]
There will be an ongoing redesign of the minting page to a new one, so I guess this is an excellent time to rethink the current flow. As from previous stages, we have been doing minor and tactical updates depending on some feedback and noticed there are a lot of leftovers which could be dropped off. Let's see how it goes 

- https://github.com/kodadot/nft-gallery/discussions/3710

**Reply by yangwao** [reply]
- think about mass minting ht @JustLuuuu 
- the easier flow of creating a bigger collection
- mass minting API 
- revamp #614

**Comment by yangwao** [comment]
chatted in an internal call and @JustLuuuu highlighted we need to update current docs, probably prioritize it rework the whole minting experience to easily onboard new people as currently, I believe there is a lot of old code, old approaches, not always perfectly handled and an itchy sketchy way to make new stuff. Plus, I've seen close-to-ready designs for mass minting

- #614 

Probably will go with @exezbcz on it once we finish collection view to some usable format to implement 

Could be also joined later with referral campaigns

- #4635

**Comment by JustLuuuu** [comment]
Just issues (and ideas) I noticed that artists struggle often with.
- not possible to see your collection in profile if it has 0 NFTs. (maybe add a category in profile for them?). Its only possible to see it during minting process
- minting, classic: struggle to notice that on top you can change from minting new collection to minting NFTs. (maybe after clicking on minting classic.. first you will get only two options (two buttons): mint nft or collection.. nothing else.. you will be able to add details (of collection or NFT) after you choose one of the options.

**Reply by yangwao** [reply]
> Just issues (and ideas) I noticed that artists struggle often with.

@JustLuuuu can have call with @exezbcz as he is in charge making UX stuff and collect feedback we will reflect in new design for minting

Meanwhile backlogged this so it's officially happening
- https://github.com/kodadot/nft-gallery/issues/4752

Classic & Simple minting will be merged for sake of simple interface and less onboarding docs to maintain
